boost-maven/boost-boosters/booster-jaxrs21/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>boost.boosters</groupId>
+    <artifactId>jaxrs</artifactId>
+    <version>2.1.M1-SNAPSHOT</version>
+
+    <dependencyManagement>
```

Comment:
Let's just not import this since we keep getting confused about how the build hangs together.   Either try to use a property at the parent POM level for the JAXRS API version (and access both from the API bom and here) so we don't repeat the prop value, or just copy-paste (and still use  a simlar-named property here).